libraries/botbuilder-dialogs/tests/skillDialog.test.js

```diff
@@ -298,7 +298,7 @@ describe('SkillDialog', function () {
             assert.strictEqual(validatedActivity.text, 'Hello SkillDialog!');
         });
 
-        it('should fail if options is falsy', function () {
+        it('should fail if options is falsy second case', function () {
```

Comment:
We could delete this second case tests, and move the line `assert.throws(() => dialog.validateBeginDialogArgs(), new TypeError('Missing options parameter'));` after the line 298 (`assert.strictEqual(validatedActivity.text, 'Hello SkillDialog!');`)